frontend/src/components/applyForm/validate.ts

```diff
@@ -0,0 +1,37 @@
+import { RJSFSchema } from "@rjsf/utils";
+import Ajv from "ajv";
+import addFormats from "ajv-formats";
```

Comment:
if we're using this it seems like it should be explicitly included in the package.json

Reply:
Added, thanks.